__tests__/app.test.js

```diff
@@ -77,42 +77,105 @@ describe("app", () => {
     });
   });
   describe("/api/review/:review_id", () => {
+    //help
     test("200: GET responds with a single review object", () => {
       return request(app)
         .get("/api/reviews/2")
         .expect(200)
         .then(({ body }) => {
-          expect(body.review).toEqual({
-            review_id: 2,
-            title: "Jenga",
-            category: "dexterity",
-            designer: "Leslie Scott",
-            owner: "philippaclaire9",
-            review_body: "Fiddly fun for all the family",
-            review_img_url:
-              "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700",
-            created_at: "2021-01-18T10:01:41.251Z",
-            votes: 5,
-          });
+          const returnedObject = body.review;
```

Comment:
please be careful editing stuff for different tickets on later branches its totally fine, it just makes my life a little more difficult reviewing :), also I think I preferred the other test as it was more explicit